Automattic/vip
# [dev-env] Tweak error handling dev-env sync

## Description

* Unifies boolean parsing logic in exec
* Adds `--force` to dev-env sync sql to avoid check if environment is up or not
* Adds check for slug presence to replace "Contact customer support" - type of error into "Use '--slug' ..." - type

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-dev-env-sync-sql.js` 
1. Should instruct you to provide env name.
